README.md

```diff
@@ -1,2 +1,18 @@
-# dfaker
-A generator of fake diabetes data.
+#Tidepool Dfaker 
+
+dfaker is a fake data generator that simulates blood glucose data over time. 
```

Comment:
Maybe change to 'blood glucose and other diabetes data over time' since that's the eventual goal?